src/file-4.xoo

```diff
@@ -0,0 +1,3 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:18571/organizations/org0/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:18571/project/issues?id=sample&branch=feature3&issues=AWE05Hw0qzxaH43k8oL1&open=AWE05Hw0qzxaH43k8oL1)

[ISSUE_KEY]: AWE05Hw0qzxaH43k8oL1